docs/source/examples/demo_notebook_for_sentence_transformer_model_training_and_uploading_to_openSearch.ipynb

```diff
@@ -0,0 +1,1377 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "fba4c02d",
+   "metadata": {},
+   "source": [
+    "# Demo Notebook for Sentence Transformer Model Training and Uploading to OpenSearch\n",
+    "This notebook provides a walkthrough step-by-step guidance for users use their synthetic queries to fine tune and train a sentence transformer model to get a semantic search model. In this notebook, you use opensearch_py_ml to accomplish the following:\n",
+    "\n",
+    "Step 0: Imports packages and Set up Client\n",
```

Comment:
Minor, make the title consistent, I searched `Imports packages and Set up Client` , but find only this line, find the later title changed `Imports ...` to `Import ...`

Reply:
Same comments for Step1 and Step2

Reply:
changing to Import